e2e/test_module.py

```diff
@@ -30,6 +31,15 @@ class TestConstants:
     ONLINE_TEST_E2E_INPUT_QUERIES = "queries"
     ONLINE_TEST_SUBDIR_PREFIX = "test_online"
 
+    EXPORT_TEST_E2E_NODES = "nodes"
+    EXPORT_TEST_E2E_RELATIONSHIPS = "relationships"
+    EXPORT_TEST_E2E_INPUT_QUERIES = "queries"
+    EXPORT_TEST_E2E_EXPORT_QUERY = "export"
+    EXPORT_TEST_E2E_IMPORT_QUERY = "import"
+    EXPORT_TEST_E2E_PLACEHOLDER_FILENAME = "_exportfile"
```

Comment:
Changed from _filename to _exportfile to not clash with _file from [PR322](https://github.com/memgraph/mage/pull/322)